### DISCUSSION TITLE
CUDA SETUP: Setup Failed!

### DISCUSSION THREAD

**Discussion by Rimuru79807980** [Linaqruf/kohya-trainer#323] [Q&A]
I have a problem with this, how do I solve it?
![‏‏لقطة الشاشة (831)](https://github.com/Linaqruf/kohya-trainer/assets/131558803/8856555d-d3aa-4e82-b4ce-d9513528a357)

**Comment by kiskejp** [comment]
I'm also getting errors on Kohya LoRA Dreambooth.
